src/utils/useSWMutation.ts

```diff
@@ -0,0 +1,12 @@
+import { useMutation } from '@apollo/client';
+import {useSnackbar} from 'notistack';
+
+
+export const useSWMutation: typeof useMutation = (a, b) => {
+    const {enqueueSnackbar} = useSnackbar();
+    return useMutation(a, {
+        onError: (error) => enqueueSnackbar(error.graphQLErrors[0].message, {variant: 'error'}),
+        fetchPolicy: 'network-only',
+        ...b,
+    });
+};
```

Comment:
Skal i ikke også have en snackbar for hvis en oprettelse af user går galt? 
Er det et aktivt valg at den ikke retunerer en snackbar hvis det går godt?
a, b? Bedre navngivning